src/cascadia/TerminalApp/App.idl

```diff
@@ -21,14 +21,14 @@ namespace TerminalApp
 
         Windows.UI.Xaml.UIElement GetRoot();
 
+		String GetTitle();

```

Comment:
please make this a _property_ (`String Title{ get; };`)

Reply:
nit: make sure it is indented properly. this is using a [tab] while our codebase uses spaces

Reply:
Changed